tests/Api/DataFixtures/ORM/authentication/customer.yaml

```diff
@@ -6,6 +6,13 @@ Sylius\Component\Core\Model\ShopUser:
         customer: "@customer_<current()>"
         username: "<current()>\\@doe.com"
         usernameCanonical: "<current()>\\@doe.com"
+    shop_user_{dave}:
```

Comment:
As this configuration is identical to the above one, we could simplify everything from lines 2 to 15 to: 
```yml
Sylius\Component\Core\Model\ShopUser:
    shop_user_{oliver, dave}:
        plainPassword: "sylius"
        roles: [ROLE_USER]
        enabled: true
        customer: "@customer_<current()>"
        username: "<current()>\\@doe.com"
        usernameCanonical: "<current()>\\@doe.com"
```

Ref. https://github.com/nelmio/alice/blob/master/doc/complete-reference.md#fixture-lists